tox.ini

```diff
@@ -43,7 +43,7 @@ deps =
     flake8-docstrings-complete>=1.0.3
     flake8-test-docs>=1.0
     isort
-    mypy==1.6.1
+    mypy==1.6.1                                                                                         
```

Comment:
nitpick but this change could be removed

Reply:
done